host/allwinner/a20.go

```diff
@@ -0,0 +1,220 @@
+	"PF1":  {"SDC0_D0", "", "JTAG_DI1"},
+	"PF2":  {"SDC0_CLK", "", "UART0_TX"},
+	"PF3":  {"SDC0_CMD", "", "JTAG_DO1"},
+	"PF4":  {"SDC0_D3", "", "UART0_RX"},
+	"PF5":  {"SDC0_D2", "", "JTAG_CK1"},
+	"PG0":  {"TS1_CLK", "CSI1_PCLK", "SDC1_CMD"},
+	"PG1":  {"TS1_ERR", "CSI1_MCLK", "SDC1_CLK"},
+	"PG2":  {"TS1_SYNC", "CSI1_HSYNC", "SDC1_D0"},
+	"PG3":  {"TS1_DVLD", "CSI1_VSYNC", "SDC1_D1"},
+	"PG4":  {"TS1_D0", "CSI1_D0", "SDC1_D2", "CSI0_D8"},
+	"PG5":  {"TS1_D1", "CSI1_D1", "SDC1_D3", "CSI0_D9"},
+	"PG6":  {"TS1_D2", "CSI1_D2", "UART3_TX", "CSI0_D10"},
+	"PG7":  {"TS1_D3", "CSI1_D3", "UART3_RX", "CSI0_D11"},
+	"PG8":  {"TS1_D4", "CSI1_D4", "UART3_RTS", "CSI0_D12"},
+	"PG9":  {"TS1_D5", "CSI1_D4", "UART3_CTS", "CSI0_D13"},
+	"PG10": {"TS1_D6", "CSI1_D6", "UART4_TX", "CSI0_D14"},
+	"PG11": {"TS1_D7", "CSI1_D7", "UART4_RX", "CSI0_D15"},
+	"PH0":  {"LCD1_D0", "", "UART3_TX", "", "PH_EINT0"},
+	"PH1":  {"LCD1_D1", "", "UART3_RX", "", "PH_EINT1"},
+	"PH2":  {"LCD1_D2", "", "UART3_RTS", "", "PH_EINT2"},
+	"PH3":  {"LCD1_D3", "", "UART3_CTS", "", "PH_EINT3"},
+	"PH4":  {"LCD1_D4", "", "UART4_TX", "", "PH_EINT4"},
+	"PH5":  {"LCD1_D5", "", "UART4_RX", "", "PH_EINT5"},
+	"PH6":  {"LCD1_D6", "", "UART5_TX", "MS_BS", "PH_EINT6"},
+	"PH7":  {"LCD1_D7", "", "UART5_RX", "MS_CLK", "PH_EINT7"},
+	"PH8":  {"LCD1_D8", "ERXD3", "KP_IN0", "MS_D0", "PH_EINT8"},
+	"PH9":  {"LCD1_D9", "ERXD2", "KP_IN1", "MS_D1", "PH_EINT9"},
+	"PH10": {"LCD1_D10", "ERXD1", "KP_IN2", "MS_D2", "PH_EINT10"},
+	"PH11": {"LCD1_D11", "ERXD0", "KP_IN3", "MS_D3", "PH_EINT11"},
+	"PH12": {"LCD1_D12", "", "PS2_SCK1", "", "PH_EINT12"},
+	"PH13": {"LCD1_D13", "", "PS2_SDA1", "SMC_RST", "PH_EINT13"},
+	"PH14": {"LCD1_D14", "ETXD3", "KP_IN4", "SMC_VPPEN", "PH_EINT14"},
+	"PH15": {"LCD1_D15", "ETXD2", "KP_IN5", "SMC_VPPPP", "PH_EINT15"},
+	"PH16": {"LCD1_D16", "ETXD1", "KP_IN6", "SMC_DET", "PH_EINT16"},
+	"PH17": {"LCD1_D17", "ETXD0", "KP_IN7", "SMC_VCCEN", "PH_EINT17"},
+	"PH18": {"LCD1_D18", "ERXCK", "KP_OUT0", "SMC_SLK", "PH_EINT18"},
+	"PH19": {"LCD1_D19", "ERXERR", "KP_OUT1", "SMC_SDA", "PH_EINT19"},
+	"PH20": {"LCD1_D20", "ERXDV", "CAN_TX", "", "PH_EINT20"},
+	"PH21": {"LCD1_D21", "EMDC", "CAN_RX", "", "PH_EINT21"},
+	"PH22": {"LCD1_D22", "EMDIO", "KP_OUT2", "SDC1_CMD", ""},
+	"PH23": {"LCD1_D23", "ETXEN", "KP_OUT3", "SDC1_CLK", ""},
+	"PH24": {"LCD1_CLK", "ETXCK", "KP_OUT4", "SDC1_D0", ""},
+	"PH25": {"LCD1_DE", "ECRS", "KP_OUT5", "SDC1_D1", ""},
+	"PH26": {"LCD1_HSYNC", "ECOL", "KP_OUT6", "SDC1_D2", ""},
+	"PH27": {"LCD1_VSYNC", "ETXERR", "KP_OUT7", "SDC1_D3", ""},
+	"PI0":  {"", "I2C3_SCL"},
+	"PI1":  {"", "I2C3_SDA"},
+	"PI2":  {"", "I2C4_SCL"},
+	"PI3":  {"PWM1", "I2C4_SDA"},
+	"PI4":  {"SDC3_CMD"},
+	"PI5":  {"SDC3_CLK"},
+	"PI6":  {"SDC3_D0"},
+	"PI7":  {"SDC3_D1"},
+	"PI8":  {"SDC3_D2"},
+	"PI9":  {"SDC3_D3"},
+	"PI10": {"SPI0_CS0", "UART5_TX", "", "PI_EINT22"},
+	"PI11": {"SPI0_CLK", "UART5_RX", "", "PI_EINT23"},
+	"PI12": {"SPI0_MOSI", "UART6_TX", "CLK_OUT_A", "PI_EINT24"},
```

Comment:
I wonder if these CLK_OUT_x are normal CLK lines. https://periph.io/x/periph/conn/gpio#pkg-constants

Reply:
Reading https://groups.google.com/forum/#!topic/linux-sunxi/zDOxAP_JJIc, it looks like yes - maybe?
What should I name these?  CLK0 and CLK1?

Reply:
Yes but that can be done in a follow up.